<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailPage</class>
 <widget class="QWidget" name="EmailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>512</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Email&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line16">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel2_2">
       <property name="text">
        <string>Outgoing mail server (SMTP):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="mailServerEdit">
       <property name="toolTip">
        <string>Enter the address of your mail server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Outgoing mail server port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="portSpin">
       <property name="toolTip">
        <string>Enter the port of the mail server</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel1_5">
       <property name="text">
        <string>Email address:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="emailEdit">
       <property name="toolTip">
        <string>Enter your email address</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel1_6">
       <property name="text">
        <string>Signature:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QTextEdit" name="signatureEdit">
       <property name="toolTip">
        <string>Enter your email signature</string>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="useTlsCheckBox">
     <property name="toolTip">
      <string>Select to use TLS</string>
     </property>
     <property name="text">
      <string>Use TLS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mailAuthenticationCheckBox">
     <property name="toolTip">
      <string>Select to authenticatate against the mail server</string>
     </property>
     <property name="text">
      <string>Mail server needs authentication</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1_15">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mailUserEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Enter your mail server username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel2_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mailPasswordEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Enter your password for accessing the mail server</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="testButton">
       <property name="toolTip">
        <string>Press to test the login data</string>
       </property>
       <property name="text">
        <string>Test Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mailServerEdit</tabstop>
  <tabstop>portSpin</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>signatureEdit</tabstop>
  <tabstop>useTlsCheckBox</tabstop>
  <tabstop>mailAuthenticationCheckBox</tabstop>
  <tabstop>mailUserEdit</tabstop>
  <tabstop>mailPasswordEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mailAuthenticationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_15</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mailAuthenticationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel2_7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mailAuthenticationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mailUserEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mailAuthenticationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mailPasswordEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
